api/restHandler/ClusterRestHandler.go

```diff
@@ -215,7 +215,7 @@ func (impl ClusterRestHandlerImpl) FindAll(w http.ResponseWriter, r *http.Reques
 	token := r.Header.Get("token")
 	clusterList, err := impl.clusterService.FindAll()
 	if err != nil {
-		impl.logger.Errorw("service err, FindAll", "err", err)
+		impl.logger.Errorw("service err, FindAllActive", "err", err)
```

Comment:
wll this change? i think this log message was update by mistake

Reply:
yes by mistake, fixed now